homeassistant/components/vasttrafik/strings.json

```
@@ -0,0 +1,100 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Set up Västtrafik",
+        "description": "Configure Västtrafik integration or add a departure board."
+      },
+      "credentials": {
+        "title": "Set up Västtrafik API",
+        "description": "Enter your Västtrafik API credentials. You can get these from the Västtrafik developer portal.",
+        "data": {
+          "key": "API Key",
+          "secret": "API Secret"
```

This also needs sentence-casing, and can be partly fixed by using a common string:

```suggestion
          "key": "[%key:common::config_flow::data::api_key%]",
          "secret": "API secret"
```